src/styles/customizations/DegenApeAcademy.css

```diff
@@ -411,6 +417,21 @@
       color: white;
       background: #1d1c1c;
       font-family: 'Raleway', 'sans-serif';
+
+      &.inFlight {
+        opacity: 0.75;
+        cursor: progress;
+        background: #bdddf445;
+      }
+    }
+
+    .searchBarWrapper {
+      color: black !important;
```

Comment:
I prefer not to use `!important`, maybe we can look for another way to apply these styles preferably in other way

Reply:
The search bar is a third party component, I couldn't override the styles in other way